robottelo/decorators/host.py

```diff
@@ -1,56 +1,35 @@
+"""Implements decorator regarding satellite host"""
 import logging
 import re
-import six
 import unittest2
 
+from backports.functools_lru_cache import lru_cache
```

Comment:
Also we should test `if PY3` using six, to take advantage of using the builtin lru_cache.

I propose creating in `robottelo.helpers`

```python
import six

if six.PY3:
    from functools import lru_cache
else:
   from repoze.lru import lru_cache
   #or
   from functools32 import lru_cache
```

With that we can simply, `from robottelo.helpers import lru_cache` in other modules